src/alpaka/AlpakaDataFormats/BeamSpotAlpaka.h

```diff
@@ -0,0 +1,29 @@
+#ifndef AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+#define AlpakaDataFormats_BeamSpot_interface_BeamSpotAlpaka_h
+
+#include "AlpakaCore/alpakaCommon.h"
+#include "DataFormats/BeamSpotPOD.h"
+
+#include <cstring>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
```

Comment:
This is more for further thought, I have an uneasy feeling about this that I'm unable to express in words. It's probably good enough for now.

If I read the template definitions correctly, as an alternative it would be possible to template the `BeamSpotAlpaka` over the Accelerator. (to repeat, I don't suggest to change this now, it is also an interesting to see how this approach pans out)

Reply:
Ok cool.
I have fully finished the RecHits with Alpaka (evth runs smoothly and results are 100% ok now in validation), and considerably advanced the pixel tracks/vertices (hoping to finish by the end of the weekkkkkk), it does not seem to have been a major issue.
In general the namespace `KOKKOS_NAMESPACE` and` ALPAKA_ACCELERATOR_NAMESPACE` are perfectly matching (not here though indeed).